account_payment_order/tests/test_payment_mode.py

```diff
@@ -9,62 +9,72 @@
 
 
 class TestPaymentMode(TransactionCase):
-    def setUp(self):
-        super(TestPaymentMode, self).setUp()
-
+    @classmethod
+    def setUpClass(cls):
+        super(TestPaymentMode, cls).setUpClass()
+        cls.env = cls.env(
+            context=dict(
+                cls.env.context,
+                mail_create_nolog=True,
+                mail_create_nosubscribe=True,
+                mail_notrack=True,
+                no_reset_password=True,
+                tracking_disable=True,
+            )
+        )
         Method_get_payment_method_information = (
             AccountPaymentMethod._get_payment_method_information
         )
 
-        def _get_payment_method_information(self):
-            res = Method_get_payment_method_information(self)
+        def _get_payment_method_information(cls):
```

Comment:
When this is something called from `setUpClass`, it should be a class method

```suggestion
        @classmethod
        def _get_payment_method_information(cls):
```